WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StatsTimeRangeBarView.swift

```diff
@@ -39,6 +41,13 @@ final class StatsTimeRangeBarView: UIView {
         container.foregroundColor = viewModel.isTimeRangeEditable ? .accent : Constants.labelColor
         configuration.attributedTitle = AttributedString(viewModel.timeRangeText, attributes: container)
 
+        if viewModel.selectedDateText != nil {
+            selectedDateLabel.text = viewModel.selectedDateText
+            selectedDateLabel.isHidden = false
+        } else {
+            selectedDateLabel.isHidden = true
+        }
```

Comment:
Nit: since `viewModel.selectedDateText` is optional, you can simply set the label with the content. The label would be hidden if the content is nil.

```suggestion
        selectedDateLabel.text = viewModel.selectedDateText
```

Reply:
Great thinking!